app/views/datagrid/_enum_checkboxes.html.erb

```diff
@@ -2,10 +2,12 @@
 Indent in this file may cause extra space to appear.
 You can add indent if whitespace doesn't matter for you
 %>
+<div class="datagrid-enum-checkboxes">
 <%- elements.each do |value, text, checked| -%>
```

Comment:
Maybe prefix checked with _ to indicate that it’s not used intentionally.
```suggestion
<%- elements.each do |value, text, _checked| -%>
```

Reply:
I don't know what to do: it is no longer used because it is not a part of UI and it doesn't make sense for user to control it. That is why V2 is determining whether it is checked or not internally.

So in idea world we want to remove it, but it causes more incompatibility because `elements` local variable ideally needs only `value` and `text`. Let me think if there is any possible way to migrate with backward compatibility here.

Reply:
Since it’s the last argument you can probably simply omit it in view: `elements.each do |value, text|`

Reply:
Is there any way we can make reference to 3rd argument generate a deprecation warning?
Alternatively, is there any way to introduce a new variable `options` having only `value` and `text` and deprecate the `elements` when referenced?